File: Courses/Bedu/DesarrolloMobileJava/Sesion4/PruebaSonidos/app/src/main/java/xyz/eliothmonroy/pruebasonidos/MainActivity.java
package xyz.eliothmonroy.pruebasonidos;

import androidx.appcompat.app.AppCompatActivity;

import android.media.MediaPlayer;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    MediaPlayer mediaPlayer;
    Button btnStop;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        btnStop=findViewById(R.id.btnStop);

        mediaPlayer=MediaPlayer.create(this,R.raw.audio);

        mediaPlayer.start();

        btnStop.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                mediaPlayer.stop();
            }
        });

    }
}

File: Courses/Bedu/KotlinIntermediate/Sesion1/HolaBedu/app/src/main/java/xyz/eliothmonroy/holabedu/MainActivity.kt
package xyz.eliothmonroy.holabedu

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        supportActionBar?.apply {
            title=getString(R.string.texto_toolbar_inicio)
        }
    }
}
